static/app/views/issueDetails/streamline/flagSeries.tsx

```diff
@@ -80,6 +48,18 @@ export default function useFlagSeries({query = {}, event}: FlagSeriesProps) {
   } = useOrganizationFlagLog({organization, query});
   const {selection} = usePageFilters();
 
+  const {
+    data: _suspectFlags,
+    isError: _isSuspectFlagsError,
+    isPending: _isSuspectFlagsPending,
+  } = useSuspectFlags({
```

Comment:
unused?

Reply:
not using the data rn to display anything, it's just used for analytics currently but need to call it here so it does something